docusaurus/docs/Android/04-compose/04-message-components/06-selected-message-overlay.mdx

```diff
@@ -1,68 +1,72 @@
-* Selected message: Shows the currently selected message. This UI is much simpler than the message item UI used to show messages in a list.
 * Message options: Shows a list of actions the user can take with the selected message, such as **delete**, **edit**, **reply to** and **start a thread** actions.
 
 Let's see how to use it!
 
 ## Usage
 
-If you're using the [`MessagesScreen`](./01-messages-screen.mdx), the `SelectedMessageOverlay` is already set up for you. To use the `SelectedMessageOverlay` in your custom screens, simply add it to your UI, like so:
+If you're using the [`MessagesScreen`](./01-messages-screen.mdx), the `SelectedMessageMenu` is already set up for you. To use the `SelectedMessageMenu` in your custom screens, simply add it to your UI, like so:
 
 ```kotlin
 // The rest of your UI
 if (selectedMessage != null) {
-    SelectedMessageOverlay(
-        // Message options
-        messageOptions = defaultMessageOptions(selectedMessage, user, listViewModel.isInThread),
-        // Message to show
+    SelectedMessageMenu(
+        // Use a Modifier to customize the appearance
+        modifier = Modifier
+            .align(Alignment.BottomCenter),
+        // Specify the message options you provide to the user
+        messageOptions = defaultMessageOptionsState(selectedMessage = selectedMessage,
+            currentUser = user,
+            isInThread = listViewModel.isInThread),
+        // Specify the message upon which the options are exerted
         message = selectedMessage,
         onMessageAction = { action ->
             // Handle message action
         },
-        onDismiss = { 
+        onDismiss = {
             // Handle dismiss
         }
     )
 }
 ```
 
-As you can see, adding the overlay is very simple. If a `selectedMessage` exists, you pass in the message options you want to expose to the user, as well as the selected message. The reactions you show are taken from the [`ChatTheme`](../02-general-customization/01-chat-theme.mdx) component and everything else required to show the component is taken care internally.
+As you can see, adding the menu is very simple. If a `selectedMessage` exists, you pass in the message options you want to expose to the user, as well as the selected message. The reactions you show are taken from the [`ChatTheme`](../02-general-customization/01-chat-theme.mdx) component and everything else required to show the component is taken care internally.
 
 The small snippet of code above produces the following UI:
 
-![Default SelectedMessageOverlay component](../../assets/compose_default_selected_message_overlay_component.png)
+![Default SelectedMessageMenu component](../../assets/compose_default_selected_message_menu.png)
 
-As you can see, the reactions you can take are shown on the top, followed by the selected message and finally by message actions.
+As you can see, the reactions you can take are shown on the top followed by message actions.
```

Comment:
let's try not to repeat ourselves too much - we can just say: "The component shows reactions from the ChatTheme on the top, followed by the message actions you passed in."

Also try to avoid using passive voice as much as possible and instead actively state which piece of code is doing what

Reply:
Changed to the suggested sentence